.github/workflows/macos.yml

```diff
@@ -67,7 +68,7 @@ jobs:
           DEVEL_PKG_NAME="$(basename package/open3d-devel-*.tar.xz)"
           echo "DEVEL_PKG_NAME=$DEVEL_PKG_NAME" >> $GITHUB_ENV
       - name: Build Open3D viewer app
-        if: ${{ matrix.CONFIG == 'OFF' }}
+        if: ${{ matrix.SHARED == 'OFF' }}
```

Comment:
@ssheorey seems like we only have `matrix.CONFIG` and `env.SHARED`. Did you mean `env.SHARED`?

Reply:
Good catch. Done.